_episodes/05-history.md

```diff
@@ -445,14 +448,23 @@ moving backward and forward in time becomes much easier.
 > >
 > > The answer is 2. 
 > > 
-> > The command `git add venus.txt` places the *current version* of `venus.txt`
-> > into the staging area. Subsequent changes to `venus.txt` are not included in the version in the 
-> > staging area. Thus, when `git add venus.txt` is issued, there is just one line in the file, and so
-> > the copy in the staging area has just one line. When the second line is appended to `venus.txt`, 
-> > the change resides in the working directory, but the copy in the staging area still has just one line.
-> > So, when the commit is issued, the file committed to the repository has just one line. The working 
-> > copy still has the second line (and `git status` will show that the file is modified). 
+> > The command `git add venus.txt` places the current version of `venus.txt` into the staging area. 
+> > The changes to the file from the second `echo` command are only applied to the working copy, 
+> > not the version in the staging area.
 > > 
+> > So, when `git commit -m "Comment on Venus as an unsuitable base"` is executed, 
+> > `venus.txt` committed to the repository is the version from the staging area and
```

Comment:
really minor nit here: but I think this might be clearer if you phrase it as 

> ... is executed, the version of `venus.txt` committed to the repository is the one from the staging area ...

rather than

> ... is executed, `venus.txt` committed to the repository is the version from the staging area

Reply:
Oh, yours is clearer.  I made the change based on your suggestion.  Thanks!